packages/esm-patient-allergies-app/src/allergies/allergies-overview.scss

```diff
@@ -6,18 +6,34 @@
   border: 1px solid $ui-03;
 }
 
-.allergiesHeader {
+.desktopHeader, .tabletHeader {
   display: flex;
   justify-content: space-between;
   align-items: center;
   padding: $spacing-04 0 $spacing-04 $spacing-05;
   background-color: $ui-background;
+
+  h4:after {
+    content: "";
+    display: block;
+    width: 2rem;
+    padding-top: 0.188rem;
+    border-bottom: 0.375rem solid $brand-teal-01;
+  }
+}
```

Comment:
I'm thinking it might be nice to factor styles out to `esm-patient-chart/esm-patient-common-lib`, if that's possible.

Reply:
Maybe if this CardHeader was a standalone component in `esm-patient-common-lib`.

Reply:
Ehhhhhh up to you. We can do it in a separate PR if you think it's worth it.

Reply:
I've worked on this here #451.